glances/plugins/glances_help.py

```diff
@@ -53,68 +61,69 @@ def generate_view_data(self):
+                ('show_hide_docker', msg_col.format('D', 'Docker')),
+                ('show_hide_top_extended_stats', msg_col.format('e', 'Top extended stats')),
+                ('show_hide_filesystem', msg_col.format('f', 'Filesystem')),
+                ('show_hide_gpu', msg_col.format('G', 'GPU')),
+                ('show_hide_ip', msg_col.format('I', 'IP')),
+                ('show_hide_tcp_connection', msg_col.format('K', 'TCP')),
+                ('show_hide_alert', msg_col.format('l', 'Alert logs')),
+                ('show_hide_network', msg_col.format('n', 'Network')),
+                ('show_hide_current_time', msg_col.format('N', 'Time')),
+                ('show_hide_irq', msg_col.format('Q', 'IRQ')),
+                ('show_hide_raid_plugin', msg_col.format('R', 'RAID')),
+                ('show_hide_sensors', msg_col.format('s', 'Sensors')),
+                ('show_hide_wifi_module', msg_col.format('W', 'Wifi')),
+                ('show_hide_processes', msg_col.format('z', 'Processes')),
+                ('show_hide_left_sidebar', msg_col.format('2', 'Left sidebar')),
+                ## Second column
+                #
+                ('show_hide_quick_look', msg_col.format('3', 'Quick Look')),
+                ('show_hide_cpu_mem_swap', msg_col.format('4', 'CPU, MEM, and SWAP')),
+                ('show_hide_all', msg_col.format('5', 'ALL')),
+                ('header_toggle', msg_header.format('TOGGLE DATA TYPE:')),
+                ('toggle_bits_bytes', msg_col.format('b', 'Network I/O, bits/bytes')),
+                ('toggle_count_rate', msg_col.format('B', 'Disk I/O, count/rate')),
+                ('toggle_used_free', msg_col.format('F', 'Filesystem space, used/free')),
+                ('toggle_bar_sparkline', msg_col.format('S', 'Quick Look, bar/sparkline')),
+                ('toggle_separate_combined', msg_col.format('T', 'Network I/O, separate/combined')),
+                ('toggle_live_cumulative', msg_col.format('U', 'Network I/O, live/cumulative')),
+                ('toggle_linux_percentage', msg_col.format('0', 'Load, Linux/percentage')),
+                ('toggle_cpu_individual_combined', msg_col.format('1', 'CPU, individual/combined')),
+                ('toggle_gpu_individual_combined', msg_col.format('6', 'GPU, individual/combined')),
+                ('toggle_short_full', msg_col.format('/', 'Process names, short/full')),
+                ('header_miscellaneous', msg_header.format('MISCELLANEOUS:')),
+                ('misc_erase_process_filter', msg_col.format('E', 'Erase process filter')),
+                ('misc_generate_history_graphs', msg_col.format('g', 'Generate history graphs')),
+                ('misc_help', msg_col.format('h', 'HELP')),
+                ('misc_accumulate_processes_by_program', msg_col.format('j', 'Accumulate processes by program')),
+                ('misc_increase_nice_process', msg_col.format('+', 'Increase nice process (need admin rights)')),
```

Comment:
it's decreasing niceness that requires elevated privileges, not increasing it, assuming you own the process

Reply:
Fixed it with bb2bbede.
(had just found it that way)